<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgHdmMap</class>
 <widget class="QDialog" name="dlgHdmMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tax code map</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="leProgram"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leHDM"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Program code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tax code</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>94</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOK">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>94</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leName"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leHDM</tabstop>
  <tabstop>leProgram</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
